src/tests/push-undelete-config-dev/expected-state.json

```diff
@@ -0,0 +1,58 @@
+{
+  "branches": [
+    {
+      "branch": {
+        "name": "Dev",
+        "description": "",
+        "isDefault": false
+      },
+      "configs": [
+        {
+          "componentId": "keboola.ex-db-mysql",
+          "name": "undeletedin\nout\nargs\nexpected-code\nexpected-state.json\nexpected-stderr\nexpected-stdout\ninitial-state.json",
+          "description": "",
```

Comment:
tento config bol v projekte `[TO DELETE]`, ale po zavolani `push` je to opat obycajny config.